Symfony/CS/Test/IntegrationSuite.php

```diff
@@ -0,0 +1,165 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Test;
+
+use Symfony\CS\FixerInterface;
+
+/**
+ * @author Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * @internal
+ */
+final class IntegrationSuite
```

Comment:
I'm not sure about the name. Is it really a suite? Isn't it a single integration test?